### ISSUE TITLE
toggle timestamp visibility 

### ISSUE THREAD

**Issue by mnods** [open-lms-open-source/moodle-mod_collaborate#13]
Hi - enhancement suggestion - we'd like an option in the plugin to not show/show the timestamp of the session. we tend to use open sessions for the duration of the course and students drop in at arranged times. the timestamp is there by default and is confusing. we've just hidden it globally but there are scenarios where it's potentially useful. Cheers

**Comment by spibullock** [comment]
Yes, we have also desired the ability to turn off this date/timestamp.  It has been a huge source of confusion for our students.  @mnods, how did you turn this off globally?  We are in Open LMS, but I didn't see this option in the site admin plugin settings for Collaborate.

**Comment by mnods** [comment]
Hi, we just hid it via theme css

**Comment by spibullock** [comment]
@mnods Oh man, that worked!!!  You have to understand, I'm not much knowledgeable about coding--but, your suggestion made me give it a try, and that did it.  I wonder if you could use the "additional CSS class" that comes with the HTML block to reverse this on a course-by-course basis?  Or maybe the global overrides all, not sure.  Again, thank you for the suggestion!

**Comment by mnods** [comment]
no worries, it's a crude way of doing it but works if you don't really mind it not showing. With us it's not needed in 95% of use cases and a teacher could just manually put in the times in the desc if needed. In theory you could add specific css underneath to say display:block for .course-idwhatever (or put them in a category and use the class name for that) but not really sustainable.
